packages/medusa/src/api/routes/admin/discounts/create-dynamic-code.ts

```diff
@@ -125,6 +125,10 @@ export class AdminPostDiscountsDiscountDynamicCodesReq {
   @IsOptional()
   usage_limit = 1
 
+  @IsNumber()
+  @IsOptional()
+  usage_limit_per_customer = 1
```

Comment:
todo: add the property to JSDocs above

Reply:
Set the default value to 0, `usage_limit` takes precedence and the database query checking for `usage_limit_per_customer` is only performed when this is explicitly set